README.md

```diff
@@ -40,6 +40,10 @@ To start all dependencies and trustification components:
 
 ``` shell
 cd deploy/compose
+
+# Required on Linux only
+sudo chcon -R -t container_file_t ./container_files/init-sso
```

Comment:
I could not get this to work. I still get the same error it the log even after running this command:
```console
[minio]
podman start -a compose_init-keycloak_1                                            
/usr/bin/bash: /init-sso/init.sh: Permission denied
```
Perhaps we could have an environment variable for the SELinux options. Something like this is compose.yaml
```yaml
- ./container_files/init-sso:/init-sso${SELINUX_VOLUME_OPTIONS}
```
And then Linux user could set this environment variable:
```console
$ env SELINUX_VOLUME_OPTIONS=':Z' podman-compose -f compose.yaml up
```
Would that work on other systems, for example mac?

Reply:
Sounds good! Could you make that modification to your PR and then I can test it?

Reply:
Absolutely, I'll re-open it and update it :+1:

Reply:
I've updated https://github.com/trustification/trustification/pull/408 now.